### DISCUSSION TITLE
Where are the uploaded PDF documents?

### DISCUSSION THREAD

**Discussion by Softbox44** [Stirling-Tools/Stirling-PDF#942] [Q&A]
Could you let me know how and where Sterling PDF stores the uploaded documents. Is it possible to retrieve and/or erase them from Sterling PDF? I love the fact that Sterling is self-hosted but need a bit of transparency regarding my documents.
many thanks

**Comment by Frooodle** [comment]
Second main paragraph of the readme 

All files and PDFs exist either exclusively on the client side, reside in server memory only during task execution, or temporarily reside in a file solely for the execution of the task. Any file downloaded by the user will have been deleted from the server by that 

So sadly no way to access the files if this is something you need

**Comment by Softbox44** [comment]
Thanks for clarifying. It would be helpful if you could dump all Pdf files
in a server folder after peocessing. Should not be hard to implement as the
files are in memory for processing as you say.
Do you think this could become a feature?
